python/lib/bidsreader.py

```diff
@@ -83,7 +83,8 @@ def load_bids_data(self):
 
         bids_config = os.environ['LORIS_MRI'] + "/python/lib/bids.json"
         exclude_arr = ['/code/', '/sourcedata/', '/log/', '.git/']
-        bids_layout = BIDSLayout(root=self.bids_dir, config=bids_config, ignore=exclude_arr)
+        force_arr   = re.compile("_annotations\.(tsv|json)$")
+        bids_layout = BIDSLayout(root=self.bids_dir, config=bids_config, ignore=exclude_arr, force_index=force_arr)
```

Comment:
```suggestion
        bids_layout = BIDSLayout(root=self.bids_dir, config=bids_config, ignore=exclude_arr, derivatives=True)
```

@laemtl see my comment on https://github.com/aces/Loris-MRI/issues/609#issuecomment-818237774

No need for the `force_index` option in the end.

Reply:
See my comment in #609. Your solution works if annotations are embedded in a derivatives folder only. It might be fine if we go this route. We need to discuss where annotations files should be stored for convenience beforehand.

Reply:
@laemtl should we discuss this tomorrow at the imaging call?